nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -143,18 +141,43 @@ def cleanup(self):
             os.remove(self.local_data_filepath)
 
     def connect_thread_func(self):
+        # Get the IP of the remote device from the fpga_config file
         remote_ip = fpga_config.get(self.fpga_name, 'ip')
 
+        # Get the SSH options from the fpga_config file
+        ssh_port = fpga_config.get(self.fpga_name, 'ssh_port')
+        ssh_user = fpga_config.get(self.fpga_name, 'ssh_user')
+
+        if fpga_config.has_option(self.fpga_name, 'ssh_pwd'):
```

Comment:
We need to handle the case where there is no password but we don't want to use ssh (eg. the DE1 default has no password)

Reply:
This should already be handled by this code.

Reply:
This does work if `ssh_key` is commented out in the config, otherwise we get `ssh_key` as an empty string. We should make it clear in docs that you can fully remove (or comment) `ssh_key` and/or `ssh_pwd`.

Reply:
This does work if `ssh_key` is commented out in the config, otherwise we get `ssh_key` as an empty string. We should make it clear in docs that you can fully remove (or comment) `ssh_key` and/or `ssh_pwd`.